tests/models/whisper/test_modeling_whisper.py

```diff
@@ -97,7 +97,7 @@ def __init__(
         hidden_dropout_prob=0.1,
         attention_probs_dropout_prob=0.1,
         max_position_embeddings=20,
-        max_source_positions=30,
+        max_source_positions=1500,
```

Comment:
Alternatively we could adjust the featurizer config to provide smaller inputs here:

https://github.com/huggingface/transformers/blob/c13e76a5e1e2322b9e6761d5870538a268830bed/tests/pipelines/test_pipelines_common.py#L167-L171

I haven't figured a reasonable set of options to do that yet though.